charts/grafana/templates/_config.tpl

```diff
@@ -85,7 +85,7 @@ download_dashboards.sh: |
 {{- range $provider, $dashboards := .Values.dashboards }}
   {{- range $key, $value := $dashboards }}
     {{- if (or (hasKey $value "gnetId") (hasKey $value "url")) }}
-  curl -skf \
+  curl {{ get $value "curlOptions" | default "-skf" }} \
```

Comment:
```suggestion
  curl {{ get $value "curlOptions" }} \
```

Reply:
NAK - this changes the previous behavior, which I did not want.

Reply:
You are right, I was a bit to tired. in the moring. Thanks for deny my proposal.